src/client/js/CommentEnabler.js

```diff
@@ -89,8 +126,13 @@ CommentEnabler.propTypes = {
 };
 
 class MethodPicker extends React.Component {
+
+
+
+
     render() {
-        return (<div>
+        return (<div className="clearfix">
+
             <div className="radio">
                 <label><input type="radio" name="optradio" value="1"/>Get comment widget id from webRunes (Recommended,
```

Comment:
Recommended, no action is required on your part.
Create your own Twitter widget. To create it you will be required to enter your Twitter account password that will be used once.